pom.xml

```diff
@@ -57,7 +57,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>credentials</artifactId>
-            <version>2.1.8</version>
+            <version>2.1.12</version>
```

Comment:
The change is available in 2.1.9, 2.1.12 forces picking a version less than a week old.